types/jquery/JQueryStatic.d.ts

```diff
@@ -2761,6 +2776,21 @@ $.post( "test.php" );
 ```
      */
     post(url_settings?: string | JQuery.UrlAjaxSettings): JQuery.jqXHR;
+    /**
+     * Load data from the server using a HTTP POST request.
+     * @param url A string containing the URL to which the request is sent.
+     * @param data A plain object or string that is sent to the server with the request.
+     * @param dataType The type of data expected from the server. Default: Intelligent Guess (xml, json, script, text, html).
+     * @see \`{@link https://api.jquery.com/jQuery.post/ }\`
+     * @since 1.0
+     * @example ​ ````Request the test.php page, send an object with data, receive a json response.
+```javascript
+$.post( "test.php", { name: 'John' }, 'json' );
+```
+     */
+    post(url: string,
```

Comment:
remove your definition and update this [post](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cd0e972f17bf5c892799692760828abe615430ee/types/jquery/JQueryStatic.d.ts#L2676-L2678) to 
```
post(url: string,
         data_success: JQuery.PlainObject | string | JQuery.jqXHR.DoneCallback | null,
         dataType: string): JQuery.jqXHR;
```